container-engine-lib/lib/backend_impls/kubernetes/object_attributes_provider/kubernetes_label_key/label_key_consts.go

```diff
@@ -28,6 +28,14 @@ const (
 
 	// As of 2022-05-17, these get attached to files artifact expansion volumes
 	userServiceGuidKeyStr = labelKeyPrefixStr + "user-service-guid"
+
+	// We create a duplicate of the enclave uuid and service uuid label key because:
+	// the logs aggregator (vector) needs the enclave uuid and service uuid label keys to create the filepath where logs are stored in persistent volume
+	// but vectors template syntax can't interpret the "kurtosistech.com/" prefix, so we can't use the existing label keys
+	logsEnclaveUuidLabelKeyStr             = "enclave_uuid"
```

Comment:
1. The other labels are using kebab-case but these are using underline_case. Is this intentional?
2. So are these label key strings are quite different, in that they'll literally *just* be `enclave_uuid` without any namespacing (no `kurtosistech.com/`)? If so, it seems worth making them even more distinct - like `logsOnlyEnclaveUuidLabelKeyStr`, making the comment much bigger, and probably adding something like `kurtosis_` in the value name so it doesn't collide with other stuff in the user's cluster

Reply:
1. underline case is intentional, asthese labels are for reading by vector and vectors template syntax example use underline case: https://vector.dev/docs/reference/configuration/template-syntax/, unsure if its a hard requirement has opposed to kebab case though, but docker backend is already using so wanted to be consistent
2. done, added kurtosis_ prefix, changed variable name of these labels, and emphasized comment